tests/pkunit_test.py

```diff
@@ -127,6 +127,13 @@ def test_pkok():
         assert 'pkunit_test.py:{}:test_pkok xyzzy 333 abc'.format(lineno) in str(e.args)
 
 
+def test_pkre_convert():
+    from pykern.pkunit import pkre
+    r = r'A'
+    for s in ('A', b'A', r'A', u'A'):
```

Comment:
As long as I'm being nitpicky.... Only two of these cases are necessary. `r` doesn't actually change the type of the string. It is akin to `f` which changes how the string is interpreted. `u` is a no-op in py3. The `r` variable is unnecessary, and the `r` on the string is unnecessary. In a non-test environment where performance might be an issue, you could compile the regex outside the loop, but that would complicate the code to no effect so better to simplify.